Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:14.18.0-stretch-slim
+LABEL maintainer="Andrey Ivanov (https://github.com/343dev)"
+
+WORKDIR /app
+
+COPY . .
+
+ENV NODE_ENV="production"
+
+RUN apt update \
+  && apt install --yes --no-install-recommends build-essential libjpeg-dev libpng-dev pkg-config \
+  && npm ci \
+  && npm link \
+  && npm cache clean --force \
```

Comment:
After installing the Optimizt dependencies, clear the npm cache, which we don't need.

This will save 15 MB.